lib/web_ui/test/text_editing_test.dart

```diff
@@ -75,6 +75,9 @@ void main() {
       expect(document.activeElement, input);
       expect(editingElement.domElement, input);
 
+      // Input is appended to the glass pane.
+      expect('${editingElement.domElement.parent.tagName}', 'FLT-GLASS-PANE');
```

Comment:
Suggestion: let's make this check less fragile. For example, this check will fail if we change the tag name of the glass pane element. It will also fail if the editing element isn't an immediate child of the glass pane.

```dart
expect(domRenderer.glassPaneElement.contains(editingElement), isTrue);
```

Reply:
done.